_posts/2023-12-05-improving-document-retrieval-with-sparse-semantic-encoders.md

```diff
@@ -0,0 +1,482 @@
+---
+layout: post
+title:  Improving document retrieval with sparse semantic encoders
+authors:
+  - zhichaog
+  - xinyual
+  - dagney
+  - yych
+  - kolchfa
+date: 2023-12-05 01:00:00 -0700
+categories:
+    - technical-posts
+meta_keywords: search relevance, neural sparse search, semantic search, semantic search with sparse encoders
+meta_description: Learn how the neural sparse framework in OpenSearch 2.11 can help you improve search relevance and optimize semantic searches with sparse encoders using just a few APIs.
+has_science_table: true
+---
+
+OpenSearch 2.11 introduced neural sparse search---a new, efficient way of semantic retrieval. In this blog post, you'll learn about using sparse encoders for semantic search. You'll find that neural sparse search reduces costs, performs faster, and improves search relevance. We're excited to share benchmarking results that show why neural sparse search is now the top-performing search method. You can even try it out by building your own search engine in just five steps. For a TLDR on benchmarking learnings, see [Key takeaways](#here-are-the-key-takeaways).
```

Comment:
We may directly emphasize the advantages in the first paragraph: good relevance and low latency.